### DISCUSSION TITLE
Recording Datetime in Metadata

### DISCUSSION THREAD

**Discussion by tetrahydra** [gopro/labs#442] [Feature Requests]
My first GoPro was 7, which I bought in 2019 and have since upgraded on every iteration until now 11. As I interchangeably use Windows and Mac with different folder managers and as the collection grew, I need to upgrade to bigger hard drives.

After several "cut" and "paste" between hard drives, now the original recording date and time have been replaced with the date a folder manager moved it. 

I would like to request a feature to add the recording date and time to the metadata.

**Comment by dnewman-gpsw** [comment]
You are in luck, it is already in our metadata since HERO10.  
CDAT field, Creation Date/Time, is the local time epoch time (seconds since Jan 1, 1970) stored within a 64-bit integer.

Use https://gopro.github.io/labs/control/metadata/ to extract this header metadata
